### PR TITLE
RUP: dar sobreturno no dinamica nominalizada en pto inicio

### PR DESCRIPTION
**Pull request by plammel**
### Requerimiento
 
https://proyectos.andes.gob.ar/browse/RUP-309
### Funcionalidad desarrollada 

1. 
2. 
3. 

### UserStory llegó a completarse

- [ ] Si
- [ ] No
- [ ] No corresponde

### Requiere actualizaciones en la base de datos

- [ ] Si
- [x] No

### Requiere actualizaciones en la API

- [ ] Si
- [x] No

### Requiere actualizaciones en andes-test-integracion

- [ ] Si
- [x] No

### CODE REVIEW

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6196](http://condorpiedra.andes.gob.ar:85/6196/mochawesome.html)
CYPRESS RUN: [5091](https://dashboard.cypress.io/projects/xr7gft/runs/5091/specs)
TEST START: 2022-04-08T15:58:44.030Z
TOTAL: 406
SUCCESS: 395
FAIL: 10
SKIPPED: 1

**Review by martinebucarey — CHANGES_REQUESTED** [review]
Si se pone un horario fuera del rango de la agenda, me deja guardar igual el sobre turno, yo lo deshabilitaría hasta que el formulario sea correcto.
https://www.loom.com/share/89d8b7b2af184dcf803fa0eb063713c4

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6249](http://condorpiedra.andes.gob.ar:85/6249/mochawesome.html)
CYPRESS RUN: [5144](https://dashboard.cypress.io/projects/xr7gft/runs/5144/specs)
TEST START: 2022-04-21T13:27:48.689Z
TOTAL: 406
SUCCESS: 374
FAIL: 31
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6253](http://condorpiedra.andes.gob.ar:85/6253/mochawesome.html)
CYPRESS RUN: [5148](https://dashboard.cypress.io/projects/xr7gft/runs/5148/specs)
TEST START: 2022-04-21T15:27:10.411Z
TOTAL: 406
SUCCESS: 395
FAIL: 10
SKIPPED: 1

**Comment by martinebucarey** [comment]
Esto sigue ocurriendo https://www.loom.com/share/89d8b7b2af184dcf803fa0eb063713c4
Revisar test

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6347](http://condorpiedra.andes.gob.ar:85/6347/mochawesome.html)
CYPRESS RUN: [5232](https://dashboard.cypress.io/projects/xr7gft/runs/5232/specs)
TEST START: 2022-05-03T19:33:14.647Z
TOTAL: 406
SUCCESS: 394
FAIL: 11
SKIPPED: 1

**Review by martinebucarey — APPROVED** [review]

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6352](http://condorpiedra.andes.gob.ar:85/6352/mochawesome.html)
CYPRESS RUN: [5237](https://dashboard.cypress.io/projects/xr7gft/runs/5237/specs)
TEST START: 2022-05-04T12:46:45.051Z
TOTAL: 406
SUCCESS: 388
FAIL: 17
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6357](http://condorpiedra.andes.gob.ar:85/6357/mochawesome.html)
CYPRESS RUN: [5242](https://dashboard.cypress.io/projects/xr7gft/runs/5242/specs)
TEST START: 2022-05-04T14:26:22.203Z
TOTAL: 406
SUCCESS: 394
FAIL: 11
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6396](http://condorpiedra.andes.gob.ar:85/6396/mochawesome.html)
CYPRESS RUN: [5281](https://dashboard.cypress.io/projects/xr7gft/runs/5281/specs)
TEST START: 2022-05-10T15:32:14.537Z
TOTAL: 406
SUCCESS: 394
FAIL: 11
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6414](http://condorpiedra.andes.gob.ar:85/6414/mochawesome.html)
CYPRESS RUN: [5296](https://dashboard.cypress.io/projects/xr7gft/runs/5296/specs)
TEST START: 2022-05-12T11:39:19.807Z
TOTAL: 406
SUCCESS: 395
FAIL: 10
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: martinebucarey  
BUILD NUMBER: [6422](http://condorpiedra.andes.gob.ar:85/6422/mochawesome.html)
CYPRESS RUN: [5304](https://dashboard.cypress.io/projects/xr7gft/runs/5304/specs)
TEST START: 2022-05-13T11:01:56.002Z
TOTAL: 406
SUCCESS: 392
FAIL: 13
SKIPPED: 1

File: src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.ts [hunk 1]
```diff
@@ -184,6 +191,48 @@ export class DinamicaFormComponent implements OnInit {
...
+                        prestacion => {
+                            this.router.navigate(['rup/ejecucion/', prestacion.id]);
+                        }
+                    );
+                } else {
+                    this.plex.info('danger', 'No fue posible crear la prestación');
+                    return EMPTY;
+                }
+
+            });
```
Comment (negro89) [inline src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.ts:222]: Recordemos que no es buena practica anidar subscribes. Se sugiere el siguiente codigo:
```suggestion
this.serviceAgenda.patch(this.agenda.id, patch).pipe(
                switchMap(agenda => {
                    const sobreTurno = agenda.sobreturnos[agenda.sobreturnos.length - 1];
                    if (sobreTurno) {
                        return this.servicioPrestacion.crearPrestacion(paciente, sobreTurno.tipoPrestacion, 'ejecucion', sobreTurno.horaInicio, sobreTurno.id);
                    } else {
                        this.plex.info('danger', 'No fue posible crear la prestación');
                        return EMPTY;
                    }
                }),
                tap(prestacion => {
                    if (prestacion) {
                        this.router.navigate(['rup/ejecucion/', prestacion.id]);
                    }
                })
            );
```

**Review by negro89 — APPROVED** [review]
El resto se ve bien!

File: src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    </paciente-buscar>
+    <paciente-listado *ngIf="pacientes && pacientes.length" [pacientes]="pacientes" [offset]="150"
+                      (selected)="onPacienteSelected($event)">
+    </paciente-listado>
+    <div *ngIf="pacientes && !pacientes.length" class="alert alert-default">
+        ¡No se encontró ningún paciente!
+    </div>
+    <fieldset>
+        <ng-container *ngIf="pacienteActivo">
+            <div class="row">
```
Comment (negro89) [inline src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.html:22]: Hubiese estado bueno plexear esto

File: src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.ts (hunk 1 above)
Comment (martinebucarey) [reply src/app/modules/rup/components/ejecucion/dacion-turno/asignar-turno.ts]: esto en que quedo??

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [6813](http://condorpiedra.andes.gob.ar:85/6813/mochawesome.html)
CYPRESS RUN: [5663](https://dashboard.cypress.io/projects/xr7gft/runs/5663/specs)
TEST START: 2022-07-07T14:16:00.351Z
TOTAL: 409
SUCCESS: 397
FAIL: 10
SKIPPED: 2

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [7107](http://condorpiedra.andes.gob.ar:85/7107/mochawesome.html)
CYPRESS RUN: [5894](https://dashboard.cypress.io/projects/xr7gft/runs/5894/specs)
TEST START: 2022-07-25T14:18:04.777Z
TOTAL: 409
SUCCESS: 397
FAIL: 11
SKIPPED: 1

**Comment by martinebucarey** [comment]
USUARIO: plammel  
BUILD NUMBER: [7111](http://condorpiedra.andes.gob.ar:85/7111/mochawesome.html)
CYPRESS RUN: [5898](https://dashboard.cypress.io/projects/xr7gft/runs/5898/specs)
TEST START: 2022-07-25T15:17:20.368Z
TOTAL: 409
SUCCESS: 397
FAIL: 11
SKIPPED: 1

**Review by AgosLizzi — APPROVED** [review]

**Comment by silviroa** [comment]
USUARIO: silviroa  
BUILD NUMBER: [7604](http://condorpiedra.andes.gob.ar:85/7604/mochawesome.html)
CYPRESS RUN: [6449](https://dashboard.cypress.io/projects/xr7gft/runs/6449/specs)
TEST START: 2022-09-12T16:05:05.725Z
TOTAL: 406
SUCCESS: 393
FAIL: 12
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: silviroa  
BUILD NUMBER: [7605](http://condorpiedra.andes.gob.ar:85/7605/mochawesome.html)
CYPRESS RUN: [6450](https://dashboard.cypress.io/projects/xr7gft/runs/6450/specs)
TEST START: 2022-09-12T16:32:30.427Z
TOTAL: 406
SUCCESS: 394
FAIL: 11
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: silviroa  
BUILD NUMBER: [7608](http://condorpiedra.andes.gob.ar:85/7608/mochawesome.html)
CYPRESS RUN: [6453](https://dashboard.cypress.io/projects/xr7gft/runs/6453/specs)
TEST START: 2022-09-13T12:02:37.343Z
TOTAL: 406
SUCCESS: 389
FAIL: 16
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: silviroa  
BUILD NUMBER: [7649](http://condorpiedra.andes.gob.ar:85/7649/mochawesome.html)
CYPRESS RUN: [6579](https://dashboard.cypress.io/projects/xr7gft/runs/6579/specs)
TEST START: 2022-09-16T14:10:06.623Z
TOTAL: 99
SUCCESS: 0
FAIL: 99
SKIPPED: 0
:warning::warning::warning::warning:
Hay archivos que no corrieron. Vuelva a intentar.

**Review by silviroa — CHANGES_REQUESTED** [review]
Reavisar los test

**Comment by negro89** [comment]
Se actualiza pr con master y corrige un error. Los tests estan ok.
image

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [7694](http://condorpiedra.andes.gob.ar:85/7694/mochawesome.html)
CYPRESS RUN: [6685](https://dashboard.cypress.io/projects/xr7gft/runs/6685/specs)
TEST START: 2022-09-27T17:36:41.502Z
TOTAL: 406
SUCCESS: 397
FAIL: 8
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [7695](http://condorpiedra.andes.gob.ar:85/7695/mochawesome.html)
CYPRESS RUN: [6686](https://dashboard.cypress.io/projects/xr7gft/runs/6686/specs)
TEST START: 2022-09-27T17:41:06.064Z
TOTAL: 406
SUCCESS: 388
FAIL: 17
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: lmlagos  
BUILD NUMBER: [7696](http://condorpiedra.andes.gob.ar:85/7696/mochawesome.html)
CYPRESS RUN: [6687](https://dashboard.cypress.io/projects/xr7gft/runs/6687/specs)
TEST START: 2022-09-27T18:03:27.189Z
TOTAL: 381
SUCCESS: 374
FAIL: 6
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: silviroa  
BUILD NUMBER: [7707](http://condorpiedra.andes.gob.ar:85/7707/mochawesome.html)
CYPRESS RUN: [6699](https://dashboard.cypress.io/projects/xr7gft/runs/6699/specs)
TEST START: 2022-09-29T15:53:18.502Z
TOTAL: 406
SUCCESS: 403
FAIL: 2
SKIPPED: 1

**Comment by silviroa** [comment]
USUARIO: mcampetella  
BUILD NUMBER: [7709](http://condorpiedra.andes.gob.ar:85/7709/mochawesome.html)
CYPRESS RUN: [6700](https://dashboard.cypress.io/projects/xr7gft/runs/6700/specs)
TEST START: 2022-09-29T16:46:19.369Z
TOTAL: 406
SUCCESS: 404
FAIL: 1
SKIPPED: 1

**Pull request merged**